oh-my-zsh/download_oh-my-zsh.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -f -x
```

Comment:
-f -x won't make it any faster

Reply:
"-f" will!

Reply:
-x is for verbose output.
-f is for forcing not to load BASH_ENV.

Reply:
Won't